File: Andy/src/level1/etc/basicString.cpp
#include <string>
#include <vector>

using namespace std;

//프로그래머스 Level 1,문자열 다루기 기본
bool solution(string s) {
    if(s.size() != 4 && s.size() != 6) return false;
    for(int i=0;i<s.size();i++)
        if(s[i] < '0' || s[i] > '9') return false;
    return true;
}

File: Andy/src/level1/etc/basicString.cs
//프로그래머스 Level 1,문자열 다루기 기본
public class Solution {
    public bool solution(string s) {
        if(s.Length != 4 && s.Length != 6) return false;
        for(int i=0;i<s.Length;i++)
            if(s[i] < '0' || s[i] > '9') return false;
        return true;
    }
}
